Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -63,55 +62,27 @@ public static int FindDigit(string equation)
             else 
             return Int32.Parse(expression);

```

Comment:
Here if the expression is such that it doesn't parse into an integer, then the code int.Parse will throw error.
So we should use int.TryParse in such scenarios.